pkg/services/ngalert/api/api_ruler_validation.go

```diff
@@ -189,8 +190,20 @@ func validateRuleGroup(
 			}
 			uids[rule.UID] = idx
 		}
+
+		ruleWithOptionals := ngmodels.AlertRuleWithOptionals{}
+		original := ruleGroupConfig.Rules[idx]
+
+		if alert := original.GrafanaManagedAlert; alert != nil {
+			isPaused := alert.IsPaused
+			if isPaused != nil {
+				ruleWithOptionals.IsPaused = *isPaused
```

Comment:
I found a bug here where `IsPaused` is overwritten on Line 205. I think we also need an integration test to make sure that `IsPaused` is updated.

Reply:
https://github.com/grafana/grafana/pull/62326/files#diff-56cde79e40d95960ccad2c25a4dd229c04b601d38967a2afb14503f97ffd50acR945